### ISSUE TITLE
Mentions autocomplete

### ISSUE THREAD

**Issue by tomastiminskas** [stakwork/sphinx-kotlin#510]
- Start caching aliases of tribe members when receiving messages on a tribe you are in
- When typing @ character, use the cached aliases to suggest and autocomplete the mention.

**Comment by tomastiminskas** [comment]
1. Create new field on Chat model: aliases - Transformable ([String] - Array of Strings) and start saving sender_alias of each messages received on a tribe
2. When typing a message, if user types an @ and the first letter after it, show a list of suggestions over the field message with the tribe aliases on that tribe matching the letters entered. Allow to select an option tapping on it or using tab key.

**Comment by tomastiminskas** [comment]
1. Create new field on Chat model: `aliases - String` (I think the best is to have a column of type String and save them separated by comma) and start saving sender_alias of each messages received on a tribe
2. When typing a message, if user types an `@` and the first letter after it, show a list of suggestions over the field message with the tribe aliases on that tribe matching the letters entered. Allow to select an option tapping on it or using tab key.

**Comment by MathRoda** [comment]
@tomastiminskas if no one working on it, i'll try to take a look at it

**Comment by tomastiminskas** [comment]
@MathRoda there is a user with name bitanon who is going to start doing these changes. So let's give him until Monday to see if he is ok with it and he will take it. Otherwise I will let you know so you can do it

**Comment by MathRoda** [comment]
@tomastiminskas sounds great.

**Comment by toadlyBroodle** [comment]
Thanks, I'll (bitanon) take a stab at it.

**Comment by toadlyBroodle** [comment]
Ok @tomastiminskas, here's my [first attempt](https://github.com/stakwork/sphinx-kotlin/compare/master...toadlyBroodle:sphinx-kotlin:master) at storing the sender_aliases from tribe chats. Problem is I apparently can't save the aliases this way, do they have to be saved to the database? I haven't used MVVM architecture much, can you clarify where I'm going wrong here? Thanks.

**Comment by toadlyBroodle** [comment]
Added second commit today that adds autocomplete popup for placeholder array of aliases. Has couple issues mentioned in commit, if you can provide feedback as to what changes need to be made that would help me out. Thanks.

**Comment by tomastiminskas** [comment]
@toadlyBroodle here is a video of the feature implemented in iOS in case you want to use as a reference regarding design


https://user-images.githubusercontent.com/2242281/206182298-c44c6736-6ddf-4cc9-9de0-fc3c9a7b1e63.MP4

**Comment by toadlyBroodle** [comment]
@tomastiminskas: Awesome, had a look at the iOS code, was somewhat helpful. 
Ended up deleting my first fork attempt and started fresh. [Second attempt](https://github.com/stakwork/sphinx-kotlin/compare/master...toadlyBroodle:sphinx-kotlin:tribe_mentions) is much more successful. 
I couldn't find the messages array you mentioned, but just grabbed all member aliases from received messages like you originally suggested. So far I've implemented the popup list of matching member aliases after typing '@' in message. 
Next I'll better constrain popup and autofill message edittext when alias is selected.
